docs/integrations/destinations/s3-migrations.md

```diff
@@ -0,0 +1,202 @@
+| time with timezone            | `Optional Record { member0: Optional Int64(logical_type=Time(Microseconds)), member1: Optional String }`      | `Optional Int64(logical_type=Time(Microseconds))`      |
+| timestamp without timezone    | `Optional Record { member0: Optional Int64(logical_type=Timestamp(Microseconds)), member1: Optional String }` | `Optional Int64(logical_type=Timestamp(Microseconds))` |
+| timestamp with timezone       | `Optional Record { member0: Optional Int64(logical_type=Timestamp(Microseconds)), member1: Optional String }` | `Optional Int64(logical_type=Timestamp(Microseconds))` |
+
+Note: the times and timestamps with timezones are converted to UTC. In parquet, the field metadata `is_adjusted_to_utc` will always be `true`.
+
+#### Time of day conversion bugfix
+
+Formerly, for the airbyte type `time_of_day_with_timezone`, [timezones were not respected when converting](https://github.com/airbytehq/airbyte/issues/43019). This has been fixed.
+
+| Input Data         | Old Output Data              | New Output Data              |
+|--------------------|------------------------------|------------------------------|
+| `"12:00:00-01:00"` | `43200000000` (12:00:00 UTC) | `46800000000` (13:00:00 UTC) |
+| `"04:00:00+02:00"` | `14400000000` (04:00:00 UTC) | `720000000` (02:00:00 UTC)   |
+| `"04:00:00+05:30"` | `14400000000` (04:00:00 UTC) | `81000000000` (22:30:00 UTC) |
+
+### Object types
+
+Formerly, objects without schemas (json-schema with `"type": "object"` but no `properties` field), and properties not listed in the `properties` field, were accumulated in `_airbyte_additional_properties`. The new behavior is
+
+* undocumented fields in object with schemas are silently dropped
+* objects without schemas are serialized into a json string
+* `_airbyte_additional_properties` is dropped entirely
+
+For example, the following input schemas and data will result in the following outputs:
+
+| Input Schema                                                          | Input Data                     | Output Schema                                                         | Output Data                          |
+|-----------------------------------------------------------------------|--------------------------------|-----------------------------------------------------------------------|--------------------------------------|
+| `{ "type": "object", "properties": { "id": { "type": "integer" } } }` | `{ "id": 1, "name": "Alice" }` | `{ "type": "object", "properties": { "id": { "type": "integer" } } }` | `{ "id": 1 }`                        |
+| `{ "type": "object" }`                                                | `{ "id": 1, "name": "Alice" }` | `{ "type": "string" }`                                                | `"{\"id\": 1, \"name\": \"Alice\"}"` |
+
+This behavior will be applied to both Avro and Parquet formats.
```

Comment:
probably worth (a) moving this to right after the `Formerly, objects without schemas...` sentence, and (b) explicitly stating that json/csv output are unaffected

Reply:
will do

Reply:
`Formerly, in both Avro and Parquet formats, objects without schemas`